docs/pages/develop/user-interface/assets.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Assets
+description: Learn about using local and remote assets.
+---
+
+Files such as images, videos, sounds, database files for SQLite, or fonts are considered **assets**. They are not part of your project's JavaScript but are part of your app.
```

Comment:
fonts too!

Reply:
I'm confused. The "fonts" are mentioned.